src/plugins/home/server/services/tutorials/tutorials_registry.ts

```diff
@@ -42,12 +79,12 @@ export class TutorialsRegistry {
     return {
       registerTutorial: (specProvider: TutorialProvider) => {
         try {
-          const emptyContext = {};
           tutorialSchema.validate(specProvider(emptyContext));
```

Comment:
NIT: what if the spec provider mutates the provided context somehow? I don't think moving it to a global const was necessary, and the perf improvement is rather minimal (unless there was another reason I missed?)

Reply:
Specproviders may not mutate, because already the specprovider is called multiple times. Once on validation (https://github.com/elastic/kibana/blob/85753fd7a1dc939f71316a5774137bdb840eeb52/src/plugins/home/server/services/tutorials/tutorials_registry.ts#L83), and then on each home page-load (https://github.com/elastic/kibana/blob/85753fd7a1dc939f71316a5774137bdb840eeb52/src/plugins/home/server/services/tutorials/tutorials_registry.ts#L74)